aws_python_utils/s3.py

```diff
@@ -3,40 +3,37 @@
 from . import logger
 from io import BytesIO, StringIO
 
-s3_client = boto3.client("s3")
-LOG = logger.getLogger("S3Util")
-
-"""
-returns tuple representing (bucket,key)
-"""
-def get_bucket_and_key_from_s3_path(s3_path):
-    res = re.search("s3://(.*?)/(.*)", s3_path)
-    if not res:
-        # try without s3 protocol
-        res = re.search("(.*?)/(.*)", s3_path)
+
+class AwsS3():
+    """AWS Secret Manager class"""
+
```

Comment:
define `__init__(self, *args)`  constructor to take params for credential configuration for s3 client

Reply:
Good call. Added to both s3 and secretsmanager classes